scripts/SREP/example/script.sh

```diff
@@ -1,7 +1,19 @@
 #!/bin/bash
 
+# Source the logging system
+source ../../LIBRARY/liblog.sh
+
+# Log a message at the INFO level
+log_info "Getting pods in openshift-monitoring namespace"
+
+# Get pods in the openshift-monitoring namespace
 oc -n openshift-monitoring get po
 
-echo "var1 is ${var1}"
+# Log a message at the DEBUG level, with a variable interpolated in the message
+var1="foo"
+log_debug "The value of var1 is ${var1}"
```

Comment:
I've tested this and it works well.
Can you specify how one can enforce debug-level logs? As I'm unable to view the above when run as default.

Reply:
you can enforce debug-level logs by setting the LOG_LEVEL environment variable to debug before executing the script. This will cause the log_debug messages to be printed in addition to log_info messages. For example
```
if [ "$LOG_LEVEL" == "debug" ]; then
  log_debug "Debug message xyz "
fi
```